llpc/unittests/CMakeLists.txt

```diff
@@ -54,7 +54,7 @@ function(llpc_add_unittest_impl test_suite test_name)
   set(outdir ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR})
```

Comment:
(I can't comment on the line outside of this change)
Can we rename `llpc_add_unittest_impl` to `add_llpc_unittest_impl` to match the public name `add_llpc_unittest` used elsewhere?

Reply:
Done